### PR TITLE
Django Tutorial Part 5: Creating our home page

### PR DESCRIPTION
**Pull request by ngocvt-0484**

### CODE REVIEW

**Comment by ngocvt-0484** [comment]
ready

File: locallibrary/catalog/templates/base_generic.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+{% load i18n %}
+<html lang="en">
+  <head>
+    {% block title %}<title>Local Library</title>{% endblock %}
```
Comment (huongnt-2545) [inline locallibrary/catalog/templates/base_generic.html]: translate title

File: locallibrary/catalog/views.py [hunk 2]
```diff
@@ -1,3 +1,26 @@
 from django.shortcuts import render
 
-# Create your views here.
+from .models import Book, Author, BookInstance, Genre
+
+def index(request):
+    """View function for home page of site."""
+
+    # Generate counts of some of the main objects
+    num_books = Book.objects.all().count()
```
Comment (huongnt-2545) [inline locallibrary/catalog/views.py]: count trực tiếp luôn, ko cần .all()
```suggestion
    num_books = Book.objects.count()
    num_instances = BookInstance.objects.count()
```

File: locallibrary/catalog/views.py [hunk 3]
```diff
@@ -1,3 +1,26 @@
...
+    num_instances = BookInstance.objects.all().count()
+
+    # Available books (status = 'a')
+    num_instances_available = BookInstance.objects.filter(status__exact="a").count()
```
Comment (huongnt-2545) [inline locallibrary/catalog/views.py:13]: C tìm hiểu thêm lookup type
https://docs.djangoproject.com/en/3.2/topics/db/queries/#field-lookups

**Review by huongnt-2545 — APPROVED** [review]

**Pull request merged**